e2e/testdata/fn-eval/exclude-fn-config-by-default/labelconfig.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: label-config
+data:
+  tier: db
```

Comment:
I remember a discussion where it was decided config provided `-fn-config` is not expected to live in the config repo. If it does, then it needs to be added to `.krmignore` file for it to be excluded. Isn't it so ?

Reply:
Yes. This fnconfig is used in pipeline and since we haven't supported it I skip this test now.